core/lib/libtomcrypt/cmac.c

```diff
@@ -82,10 +83,14 @@ static const struct crypto_mac_ops ltc_omac_ops = {
 	.copy_state = ltc_omac_copy_state,
 };
 
-TEE_Result crypto_aes_cmac_alloc_ctx(struct crypto_mac_ctx **ctx_ret)
+static TEE_Result crypto_common_cmac_alloc_ctx(struct crypto_mac_ctx **ctx_ret,
```

Comment:
fix indentation.
could rename `_cmac_alloc_ctx()`, shorter.

Reply:
I fixed indentation as requested.
I decided not to shorten the name of the function this time though as it's static anyway and I can't see any obvious advantage to make it shorter tbh.
Sure I'm OK with changing it if you really think it makes sense.